components/clp-package-utils/clp_package_utils/general.py

```diff
@@ -645,6 +645,14 @@ def is_retention_period_configured(clp_config: CLPConfig) -> bool:
     return False
 
 
+def validate_presto_config(clp_config: CLPConfig) -> None:
+    if clp_config.package.query_engine == QueryEngine.PRESTO:
+        if clp_config.presto is None:
+            raise ValueError(
+                f"Presto config must be set when query_engine is `{clp_query_engine}`"
+            )
```

Comment:
Nit: I got similar comments many times from Zhihao and Kirk, so let's go with less indent.
```suggestion
    if clp_config.package.query_engine == QueryEngine.PRESTO and clp_config.presto is None:
        raise ValueError(
            f"Presto config must be set when query_engine is `{clp_query_engine}`"
        )
```